scripts/cleanup-test-images.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+if [ "$#" -ne 4 ]; then
+  echo "Usage: $0 <org> <repo> <package-name> <github-token>"
+  exit 1
+fi
+
+ORG="$1"
+REPO="$2"
+PACKAGE_NAME="$3"
+GITHUB_TOKEN="$4"
+
+API_URL="https://api.github.com/orgs/$ORG/packages/container/$PACKAGE_NAME/versions?per_page=100"
+
+CURRENT_DATE=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+
+response=$(curl -L \
+  -H "Accept: application/vnd.github+json" \
+  -H "Authorization: Bearer $GITHUB_TOKEN" \
+  -H "X-GitHub-Api-Version: 2022-11-28" \
+  "$API_URL")
+
+echo "$response" | jq -r \
+  --arg current_date "$CURRENT_DATE" \
+  'map(select(.created_at < $current_date)) | .[] | .id' | while read version_id; do
```

Comment:
Only a minor comment but, doesn't this mean that it will delete all images created before the current date? Effectively this translates into all images and the date filter could be removed, don't you think?